tensorflow/workspace2.bzl

```diff
@@ -597,10 +597,11 @@ def _tf_repositories():
     )
 
     # WARNING: make sure ncteisen@ and vpai@ are cc-ed on any CL to change the below rule
+    # fix for https://github.com/protocolbuffers/upb/issues/300 - using grpc v1.36.4
     tf_http_archive(
         name = "com_github_grpc_grpc",
-        sha256 = "b956598d8cbe168b5ee717b5dafa56563eb5201a947856a6688bbeac9cac4e1f",
-        strip_prefix = "grpc-b54a5b338637f92bfcf4b0bc05e0f57a5fd8fadd",
+        sha256 = "8eb9d86649c4d4a7df790226df28f081b97a62bf12c5c5fe9b5d31a29cd6541a",
```

Comment:
Checksum is `89bab58f7bd36f2826b0bde92ea5668324642fe281d636dd18025354586cb764`

Reply:
Hey @bhack 
I double checked and it shows me 89bab...
`$ sha256sum v1.36.4.tar.gz`
`8eb9d86649c4d4a7df790226df28f081b97a62bf12c5c5fe9b5d31a29cd6541a  v1.36.4.tar.gz`

Reply:
Mhh.. In the CI log I see:
```
WARNING: Download from https://storage.googleapis.com/mirror.tensorflow.org/github.com/grpc/grpc/archive/3e53dbe8213137d2c731ecd4d88ebd2948941d75.tar.gz failed: class com.google.devtools.build.lib.bazel.repository.downloader.UnrecoverableHttpException GET returned 404 Not Found
WARNING: Download from https://github.com/grpc/grpc/archive/3e53dbe8213137d2c731ecd4d88ebd2948941d75.tar.gz failed: class com.google.devtools.build.lib.bazel.repository.downloader.UnrecoverableHttpException Checksum was 89bab58f7bd36f2826b0bde92ea5668324642fe281d636dd18025354586cb764 but wanted 8eb9d86649c4d4a7df790226df28f081b97a62bf12c5c5fe9b5d31a29cd6541a
```

Reply:
`3e53dbe8213137d2c731ecd4d88ebd2948941d75.tar.gz` is not `v1.36.4.tar.gz` Can you point on `v1.36.4.tar.gz`?

Reply:
updated.